modules/images-using-customizing-s2i-images.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * openshift_images/using_images/using-images-source-to-image.adoc
+
+[id="images-using-customizing-s2i-images_{context}"]
+= Customizing source-to-image images
+
+Source-to-image (S2I) builder images normally include `assemble` and `run` scripts, but the default behavior of those scripts may not be suitable for all users. This topic covers a few approaches for customizing the behavior of an S2I builder that includes default scripts.
+
+[id="images-using-customizing-s2i-images-scripts-embedded_{context}"]
+== Invoking Scripts Embedded in an Image
+
+Typically, builder images provide their own version of the S2I scripts that cover the most common use-cases. If these scripts do not fulfill your needs, S2I provides a way of overriding them by adding custom ones in the `.s2i/bin` directory. However, by doing this you are completely replacing the standard scripts. In some cases this is acceptable, but in other scenarios you may prefer to execute a few commands before (or after) the scripts while retaining the logic of the script provided in the image. In this case, it is possible to create a wrapper script that executes custom logic and delegates further work to the default script in the image.
+
+To determine the location of the scripts inside of the builder image, look at the value of `io.openshift.s2i.scripts-url` label. Use `podman inspect`:
+
+[source,terminal]
+----
+$ podman inspect --format='{{ index .Config.Labels "io.openshift.s2i.scripts-url" }}' openshift/wildfly-100-centos7
```

Comment:
@adambkaplan Should we replace all images from docker hub?

Reply:
It looks like Wildfly has moved to Quay.io (https://quay.io/search?q=wildfly-100-centos). Should we update/change the image referenced?

Reply:
Yes - quay.io is the new home for Wildfly images.